cgi/product_image_upload.pl

```diff
@@ -293,8 +296,8 @@
 
 		if ($User{moderator}) {
 			$product_ref = retrieve_product($product_id);
-			$image_data_ref->{uploader} = $product_ref->{images}{$imgid}{uploader};
-			$image_data_ref->{uploaded} = $product_ref->{images}{$imgid}{uploaded_t};
+			$image_data_ref->{uploader} = $product_ref->{images}{uploaded}{$imgid}{uploader};
+			$image_data_ref->{uploaded} = display_date($product_ref->{images}{uploaded}{$imgid}{uploaded_t});
```

Comment:
```suggestion
			$image_data_ref->{uploaded} = display_date_iso($product_ref->{images}{uploaded}{$imgid}{uploaded_t});
```
A normalized ISO version might be a good idea for cases where the date is returned as structured data?

Reply:
This one is only for internal use, for the web product edit form, for moderators, and it is displayed as-is. We could change it, but then we would need to change the JS too..